sirmordred/task_collection.py

```diff
@@ -140,6 +143,20 @@ def execute(self):
         print("Collection for {}: finished after {} hours".format(self.backend_section,
                                                                   spent_time))
 
+        data_retention_window = cfg['general']['retention_days']
+        if data_retention_window:
+            self.retain_raw_data(data_retention_window)
+
+    def retain_raw_data(self, days_to_retain):
```

Comment:
This only removes data from the raw. I think we should be cautious and remove data from the enriched indexes too.

Reply:
I added the method also after the enrichment phase. 
The main problem concerns the studies executed with ceres because they can create other indexes and mordred doesn't parse the study params (this was a choice to avoid tweaking the setup.cfg and delegate the handling of the params to the study).